### ISSUE TITLE
Publish on crates.io, so documentation will be available on docs.rs

### ISSUE THREAD

**Issue by tikhono** [mir-protocol/plonky2#856]

**Comment by Nashtare** [comment]
I believe this can be closed.
Btw @dlubarov I think plonky2 0.1.3 is like 5 months old now, it would be nice to update the crate version with the recent features added.

**Comment by dlubarov** [comment]
@Nashtare yeah good point, it should be updated now (thanks @npwardberkeley)
